### PR TITLE
feat(ui): add Rinkeby testnet and Mainnet

### PR DESCRIPTION
**Pull request by rudolfs**
Backporting these changes from the [orgs](https://github.com/radicle-dev/radicle-upstream/pull/1813) PR to make it easier to review.

We'll need the Rinkeby testnet to test Orgs and Gnosis Safe functionality and Mainnet for our production builds.

### CODE REVIEW

File: ui/src/wallet.ts [hunk 1]
```diff
@@ -65,6 +65,16 @@ function getProvider(
...
       );
+    case ethereum.Environment.Rinkeby:
+      return new ethers.providers.InfuraProvider(
+        "rinkeby",
+        "de5e2a8780c04964950e73b696d1bfb1"
+      );
+    case ethereum.Environment.Mainnet:
+      return new ethers.providers.InfuraProvider(
+        "mainnet",
+        "7a19a4bf0af84fcc86ffb693a257fad4"
```
Comment (geigerzaehler) [inline ui/src/wallet.ts:76]: Could you add a comment saying that you have access to this infura token?

Comment (rudolfs) [reply ui/src/wallet.ts:77]: 798a7a9b8f4f1f8ae4cf5acb5ef31ee8cef16959

Comment (geigerzaehler) [reply ui/src/wallet.ts:77]: Could you add the same comment to the Rinkeby token :innocent:?

Comment (rudolfs) [reply ui/src/wallet.ts:77]: I can't find anymore who's account it was 😬

**Review by geigerzaehler — APPROVED** [review]

**Pull request merged**